src/svd.cc

```diff
@@ -236,7 +246,9 @@ void svd(
         U2.insertLocalTiles();
 
         // Reduce band to bi-diagonal.
+        Timer t_tb2db;
```

Comment:
Typo: should be `tb2bd` (triangular band to bidiagonal). Also in a few other places:
```
pangolin ~/Documents/slate> git grep tb2db
src/svd.cc:        Timer t_tb2db;
src/svd.cc:        timers[ "svd::tb2bd" ] = t_tb2db.stop();
test/test_svd.cc:        params.time5.name( "tb2db (s)" );
test/test_svd.cc:            params.time5() = slate::timers[ "svd::tb2db" ];
```